docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,130 @@
+
+### Filter on column headers [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+This feature allows users to quickly filter the data without needing any additional menu and can be activated using the `unstable_headerFilters` prop.
+
+The new filter fields are displayed below the headers and are synchronized with the filter panel, but for those seeking more simplicity, disable the default filter panel using `disableColumnFilter` prop and set filters to use only the default operator.
+You can also customize the header filter cell for a specific column using the `renderHeaderFilter` method of `GridColDef`
+
+For more details on how to use and customize the filters, check out its [documentation page](https://mui.com/x/react-data-grid/filtering/header-filters/).
+
+### **Performance Enhancements: Smoother Scrolling and Expanding Rows**
+
+The Data Grid received significant performance improvements in MUI X v6, particularly concerning the data grid's scrolling efficiency and the expansion of children's rows.
+
+These performance upgrades are part of our ongoing commitment to provide end-users with a superior experience.
+We always strive to ensure that MUI X remains performant and efficient, particularly when dealing with large and complex data.
+
+## Charts - alpha version 🎁
+
+We are delighted to announce the upcoming expansion of MUI X– a brand-new suite of components specifically designed for building and easily customizing appealing charts.
+
+With MUI X Charts, you can choose from a wide range of chart types, including line charts, bar charts, pie charts, area charts, scatter plots, and more. Each chart type is thoughtfully crafted with attention to detail, ensuring that the visual representations are not only aesthetically pleasing but also highly effective in conveying complex data.
+
+And it ships with a couple of rad palettes, specially crafted by our designers!
+
+Checkout a video highlighting some of our Charts!
+
+<video width="400" height="400" preload="metadata" controls>
+  <source src="/static/blog/mui-x-mid-v6-features/charts_final_v1.mov" type="video/mp4">
+</video>
+
+As we progress toward the stable version, we're committed to enhancing the overall experience for developers and users, so your feedback is fundamental!
+
+We will also steadily expand our portfolio with new chart types; the Pie Chart is next on our list for the first stable version.
+However, if there's a specific chart visualization you'd like us to prioritize, we encourage you to upvote the respective issue on [GitHub](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+charts%22+label%3A%22waiting+for+%F0%9F%91%8D%22). Your input can directly influence our development schedule, so don't hesitate to let us know what matters most to you!
+
+[Get started with charts now!](https://mui.com/x/react-charts/)
+
+## Discord
```

Comment:
I think I'd also move the discord to after "What's next", perhaps under ## Feedback would help connect the topic with the blog post.

Reply:
One thing that struck me is that having the Discord announced on a blog post centered around X makes me think it's an X-specific thing, which is not! Not sure whether this resonates but I wondered if we should either expand the post scope to cover MUI as a whole or keep it as is (could be not a big deal).

Reply:
@danilo-leal You're totally right. I think we should separate this as a standalone blog